unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldBridge/BuilderInWorldBridge.cs

```diff
@@ -16,8 +16,8 @@
 /// </summary>
 public class BuilderInWorldBridge : MonoBehaviour
 {
-    public event Action OnPublishOK;
-    public event Action<string> OnPublishKO;
+    public event Action OnPublishOk;
+    public event Action<string> OnPublishKo;
```

Comment:
I'm still not convinced by this naming as its very hard to understand. What can be done about this? Maybe ditch the `Action` field and rename it to `OnPublishFinished` or something like that?